scripts/pylib/twister/twisterlib/twister_main.py

```diff
@@ -131,6 +131,12 @@ def main(options: argparse.Namespace, default_options: argparse.Namespace):
         logger.error(f"{e}")
         return 1
 
+    def check_platform(platform, platform_list):
```

Comment:
code duplication, the same method already exists in testplan.py

Reply:
addressed